mcs/class/Makefile

```diff
@@ -105,6 +105,8 @@ monodroid_dirs := \
 	$(mobile_common_dirs)	\
 	Mono.CompilerServices.SymbolWriter	\
 	Mono.Btls.Interface	\
+	System.Runtime.Serialization.Formatters.Soap	\
+	System.Runtime.Remoting	\
```

Comment:
We need SRR if we want AppDomains to work beyond trivial examples as most forms of cross-domain communications need it.

At this stage, it's ok if XA doesn't ship those two assemblies.

Additionally, given that this assembly is loaded by corlib using reflection, there's further work in tooling to make it possible to ship it without impacting our users.